specification/schemas/InvoiceRelationship.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "object",
+  "required": [
+    "data"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "data": {
+      "type": "object",
+      "required": [
+        "type"
```

Comment:
`id` should also be required in a relationship, and the `related` link should be added as well.

Reply:
Invoice does not have a endpoint yet for the related link

Reply:
That was added in v0.21.2